### PR TITLE
Nvcr 66 update readme file

### PR DESCRIPTION
**Pull request by KateSydor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-`npm install @material/web`
-
-`npm install --save-dev @web/dev-server`
-
-`npx web-dev-server --node-resolve`
+#### The site was created to enter accounting for turnover, control the amount of products in the warehouse, calculate customer debts, the total amount of profit received, create invoices (on the site and export in PDF format, for printing)
+To clone repo use command:\
+`git clone https://github.com/student-project-lnu-2022/invocer.git` \
+Go to directory invoicer_project, open terminal and run next commands:\
+`pip install djangorestframework` \
```
Comment (maxchv) [inline README.md:7]: it is better:

```
cd invoicer_project
pip install -r requirements.txt
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+`pip install django-cors-headers` \
+`pip install djangorestframework-simplejwt` \
+`pip install reportlab` \
+To run the server use command:\
+`python manage.py runserver`
```
Comment (maxchv) [inline README.md:12]: Here you missed `makemigrations `and `migrarte `command

File: README.md (hunk 1 above)
Comment (KateSydor) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (KateSydor) [reply README.md:13]: done

**Review by maxchv — APPROVED** [review]

**Review by KateYatsiuk — APPROVED** [review]

**Review by Vitalik2705 — APPROVED** [review]

**Review by NadiiaNanivska — APPROVED** [review]

**Review by dmitru4ok — APPROVED** [review]

**Pull request merged**